app/src/main/res/layout/activity_main.xml

```diff
@@ -93,4 +93,18 @@
         app:layout_constraintStart_toStartOf="@id/humidity"
         app:layout_constraintTop_toBottomOf="@id/humidity" />
 
+    <Button
+        android:id="@+id/forecast_btn"
+        android:layout_width="200dp"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="24dp"
+        android:backgroundTint="#A6A6A6"
+        android:onClick="forecastData"
```

Comment:
While this works it is bad practice since it mixes declarative UI code (i.e. XML) with view logic (i.e. `setOnClickListener`).